app/models/audit/AuditTaskTable.scala

```diff
@@ -449,7 +451,7 @@ object AuditTaskTable {
         StreetEdgeAssignmentCountTable.incrementAssignment(e.streetEdgeId)
         NewTask(e.streetEdgeId, e.geom, e.x1, e.y1, e.x2, e.y2, timestamp, completed=false)
       case _ =>
-        selectANewTask // The list is empty for whatever the reason. Probably the user has audited all the streets in the region
+        selectANewTask(userId) // The list is empty for whatever the reason. Probably the user has audited all the streets in the region
     }
```

Comment:
This method tries to get a new task in this region that the user has not audited, and if it finds none, it needs to get a new task from some other region. selectANewTask is an overloaded method, where no userId is taken in for anon users, but this method was erroneously calling the one for anon users. That version of the selectANewTask method does not filter out tasks that a user has already completed, so a user could have been assigned a task that they already did.